quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -116,16 +109,21 @@ private static boolean hasAutoBuildOption(List<String> cliArgs) {
         return cliArgs.contains(AUTO_BUILD_OPTION_LONG) || cliArgs.contains(AUTO_BUILD_OPTION_SHORT);
     }
 
-    private static boolean requiresReAugmentation(CommandLine cmd) {
+    public static boolean requiresReAugmentation(CommandLine cmd) {
         if (hasConfigChanges()) {
-            cmd.getOut().println("Changes detected in configuration. Updating the server image.");
+            if (cmd != null) {
```

Comment:
can we merge this with the surrounding if()?

Reply:
Done